src/frontend/frontend/file.js

```diff
@@ -384,7 +380,7 @@ angular.module('frontend-app')
         self.runFile = function() {runWhenSaved(function () {
           self.killProgram().then(function() {
             self.console.clear();
-            self.project.run(self.question, "question", self.runnerFile, self.contents, false)
+            self.project.run(self.question, "question", self.file, self.contents, false)
```

Comment:
Can we omit file entirely, as it's kind of confusing to see it here when we don't actually depend on it (we really only depend on the question now)